samples/document-snippets/instance.js

```diff
@@ -74,44 +66,30 @@ const snippets = {
       location: 'us-central1-b',
       storage: 'hdd',
     };
-    instance
-      .createCluster(clusterId, options)
-      .then(result => {
-        const newCluster = result[0];
-        // const operations = result[1];
-        // const apiResponse = result[2];
-      })
-      .catch(err => {
-        // Handle the error.
-      });
+    const [newCluster,operations,apiResponse] = await instance
+      .createCluster(clusterId, options);
     // [END bigtable_create_cluster]
   },
 
-  createAppProfile: (instanceId, clusterId, appProfileId, callback) => {
+  createAppProfile: async (instanceId, clusterId, appProfileId, callback) => {
     // [START bigtable_create_app_profile]
     const Bigtable = require('@google-cloud/bigtable');
     const bigtable = new Bigtable();
     const instance = bigtable.instance(instanceId);
-
+    const util = require('util');
```

Comment:
I believe this is not needed as per below.

Reply:
I promisify callback function using util, to use async and await.

Reply:
[`instance.createAppProfile()`](https://github.com/googleapis/nodejs-bigtable/blob/9e07093aa813f99ced5519de09bcaa6b7faeb0c8/src/instance.js#L156) should already be promisified in the `instance.js` module

https://github.com/googleapis/nodejs-bigtable/blob/9e07093aa813f99ced5519de09bcaa6b7faeb0c8/src/instance.js#L766-L768

Reply:
[`instance.createAppProfile()`](https://github.com/googleapis/nodejs-bigtable/blob/9e07093aa813f99ced5519de09bcaa6b7faeb0c8/src/instance.js#L156) should already be promisified in the `instance.js` module

https://github.com/googleapis/nodejs-bigtable/blob/9e07093aa813f99ced5519de09bcaa6b7faeb0c8/src/instance.js#L766-L768

Reply:
removed!